src/FeatureIntroLearnMoreLink/index.tsx

```diff
@@ -0,0 +1,52 @@
+/** @jsx jsx */
+import * as React from "react";
+import * as typography from "../typography";
+import { jsx } from "@emotion/core";
+import { colors } from "../colors";
+import { useFeatureIntroControlInternalContext } from "../shared/FeatureIntroControlContext";
+
+/**
+ * Component to render Feature Intro learn more button.
+ *
+ * If this component is rendered in the children of `FeatureIntroControl`, then
+ * `FeatureIntroControl` will render this element in it's layout. Otherwise, it's
+ * rendered as-is.
+ */
+export const FeatureIntroLearnMoreLink: React.FC<React.HTMLProps<
+  HTMLAnchorElement
+>> = ({ className, style, id, href, rel, target }) => {
```

Comment:
hmm yeah, we would have to do some work around to make it stable.
Perhaps this could accept an `as` prop where someone could do `as={<a ...any prop they want />}` that would also allow us to make it a react-router link if we need that in the future